includes/payment-methods/class-wc-stripe-express-checkout-element.php

```diff
@@ -324,7 +328,7 @@ public function display_express_checkout_button_html() {
 		}
 
 		?>
-		<div id="wc-stripe-express-checkout-wrapper" style="margin-top: 1em;clear:both;display:none;">
+		<div id="wc-stripe-express-checkout-wrapper" style="margin-top: 1em;clear:both;">
```

Comment:
Curious why you removed the `displey:none;` here?
I think the default should always be display none and then show the elements via JS.

WooPayments just made this change in a recent fix: https://github.com/Automattic/woocommerce-payments/commit/8f2b1ac66610479282b375078ba00b5aa4d1dfe9

Reply:
Good catch, Matt! I reverted this change here and followed the implementation you liked. It seems to be working fine now 👍